data.json

```
@@ -175,6 +175,15 @@
             ],
             "description": "A music programming language for musicians. 🎶"
         },
+        {
+            "name": "Metabase",
+            "link": "https://github.com/metabase/metabase",
+            "label": "good first issue",
+            "technologies": [
+                "Clojure"
```

This could also have TypeScript added to it as the repo is largely Clojure and TypeScript.

```suggestion
                "Clojure",
                "TypeScript"
```